sdk/testutil/types.go

```diff
@@ -9,4 +9,5 @@ type TestingTB interface {
 	Logf(format string, args ...interface{})
 	Name() string
 	Fatalf(fmt string, args ...interface{})
+	Helper()
```

Comment:
honest question, why we needed to add `Helper` to this interface?

Reply:
I switched `defaultServerConfig` to use `GetN` instead of `Take`, which requires this interface to have `Helper()`